x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/components/total_charts.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useValues } from 'kea';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiPageContentBody,
+  EuiTitle,
+  EuiText,
+} from '@elastic/eui';
+
+import { EuiButton } from '../../../../shared/react_router_helpers';
+
+import { ENGINE_ANALYTICS_PATH, ENGINE_API_LOGS_PATH, getEngineRoute } from '../../../routes';
+import { TOTAL_QUERIES, TOTAL_API_OPERATIONS } from '../../analytics/constants';
+import { VIEW_ANALYTICS, VIEW_API_LOGS, LAST_7_DAYS } from '../constants';
+
+import { EngineLogic } from '../../engine';
+import { EngineOverviewLogic } from '../';
+
+export const TotalCharts: React.FC = () => {
+  const { engineName } = useValues(EngineLogic);
+  const engineRoute = getEngineRoute(engineName);
+
+  const {
+    // startDate,
+    // endDate,
+    // queriesPerDay,
+    // operationsPerDay,
+  } = useValues(EngineOverviewLogic);
+
+  return (
+    <EuiFlexGroup>
+      <EuiFlexItem>
+        <EuiPageContent data-test-subj="TotalQueriesChart">
+          <EuiPageContentHeader responsive={false}>
+            <EuiPageContentHeaderSection>
+              <EuiTitle size="xs">
+                <h2>{TOTAL_QUERIES}</h2>
+              </EuiTitle>
+              <EuiText size="s" color="subdued">
+                {LAST_7_DAYS}
+              </EuiText>
+            </EuiPageContentHeaderSection>
+            <EuiPageContentHeaderSection>
+              <EuiButton to={engineRoute + ENGINE_ANALYTICS_PATH} size="s">
+                {VIEW_ANALYTICS}
+              </EuiButton>
+            </EuiPageContentHeaderSection>
+          </EuiPageContentHeader>
+          <EuiPageContentBody>
+            TODO: Analytics chart
+            {/* <EngineAnalytics
+              data={[queriesPerDay]}
+              startDate={new Date(startDate)}
+              endDate={new Date(endDate)}
+            /> */}
```

Comment:
To be clear I'm not 100% on if this will translate perfectly to EuiChart as I haven't spiked that out much, I'm just leaving this in as reference from our old code